Mage.Sets/src/mage/sets/dragonsmaze/BlazeCommando.java

```diff
@@ -102,10 +99,10 @@ public BlazeCommandoTriggeredAbility copy() {
     public void reset(Game game) {
         /**
          * Blaze Commando's ability triggers each time an instant or sorcery spell you control
-         * deals damage (or, put another way, the number of times the word “deals” appears in
+         * deals damage (or, put another way, the number of times the word "deals" appears in
          * its instructions), no matter how much damage is dealt or how many players or permanents
-         * are dealt damage. For example, if you cast Punish the Enemy and it “deals 3 damage to
-         * target player and 3 damage to target creature,” Blaze Commando's ability will trigger
+         * are dealt damage. For example, if you cast Punish the Enemy and it "deals 3 damage to
+         * target player and 3 damage to target creature," Blaze Commando's ability will trigger
```

Comment:
Using "Smart Quotes" causes character encoding issues with some IDEs (like Eclipse).  Since these are only in comments anyways, I'm updating them to simple quotes to ensure compatibility.